specifications/config.json

```diff
@@ -0,0 +1,127 @@
+{
+  "client": {
+    "apiKey":{
+      "file":"",
+      "id":"",
+      "secret":""
+    },
+    "cacheManager":{
+      "defaultTtl": 300,
+      "defaultTti": 300,
+      "caches": {
+        "account": {
+          "ttl": 300,
+          "tti": 300
+        }
+      }
+    },
+    "baseUrl": "https://api.stormpath.com/v1",
+    "connectionTimeout": 30,
+    "authenticationScheme": "SAUTHC1", // or "BASIC"
+    "proxy":{
+      "port": "",
+      "host": "",
+      "username": "",
+      "password": ""
+    }
+  },
+  "application": {
+    "name": "test",
+    "href": "https://..."
+  },
+  "web": {
+    "oauth2":{
+      "enabled": false,
+      "uri": "/oauth/token",
+      "password":{
+        "enabled": true,
+        "accessToken":{
+          "ttl": 3600
+        }
+      },
+      "client_credentials":{
+        "enabled": true,
+        "accessToken":{
+          "ttl": 3600
+        }
+      }
+    },
+    "accessTokenCookie":{
+      "name": "access_token",
+      "ttl": 3600,
+      "tti": 900,
+      "httpOnly": true,
+      "secure": null,
+      "path": "/",
+      "domain": null
+    },
+    "register": {
+      "enabled": false,
+      "uri": "/register",
+      "nextUri": "/",
+      "autoAuthorize": false,
+      "fields": [
```

Comment:
I like the `fieldOrder` idea.

JS linters puke on invalid JSON, and many test suites (including our Node SDK) will run files through a linter as part of the tests, so this will fail.  And it also means that these comments are showing red in my text editor, not fun.